data/net/minecraft/world/entity/LivingEntity.mapping

```diff
@@ -258,12 +265,23 @@ CLASS net/minecraft/world/entity/LivingEntity
 		ARG 1 fluidTag
 	METHOD kill ()V
 		COMMENT Called by the /kill command.
+	METHOD knockback (DDD)V
+		ARG 1 x
+		ARG 3 y
+		ARG 5 z
```

Comment:
It seems like the first parameter is the strength of the knockback, while the remaining two parameters are the X and Z components for the vector of the knockback.
```suggestion
	METHOD knockback (DDD)V
		ARG 1 strength
		ARG 3 x
		ARG 5 z
```